coord/src/module.c

```diff
@@ -1574,6 +1590,14 @@ static int searchResultReducer(struct MRCtx *mc, int count, MRReply **replies) {
     rm_free(rCtx.cachedResult);
   }
 
+  // If we timed out on strict timeout policy, return a timeout error
+  if (req->timeout != 0
+      && (((double)(clock() - req->initClock) / CLOCKS_PER_MILLISEC) > req->timeout)
+      && RSGlobalConfig.requestConfigParams.timeoutPolicy == TimeoutPolicy_Fail) {
```

Comment:
Replace order of operands

Reply:
Call it from the lopp that does the processing above (per each number of iterations)